.github/workflows/hosted.yml

```diff
@@ -0,0 +1,290 @@
+name: antlr4
+
+on:
+  push:
+    branches: [ master, dev, hostedci ]
```

Comment:
Is `hostedci` branch required? It looks temporary.

Reply:
I used this branch for local testing in my fork without having to modify dev branch. It plays no role in upstream repo.